### PR TITLE
here is my docs copyedit

### PR DESCRIPTION
**Pull request by chrisdavidmills**
I had a few comments too:

## properties.md

* If it is an enum (appropriate for long-hand properties), this is what the percentage of the property refers to.
* If it is an array (appropriate for short-hand properties), these are the properties the percentages refer to.

-- I find these definitions a bit confusing. Percentages are percentages. The value in the example above is "no" - how does a percentage refer to "no"? Could we try to rephrase this to make it a bit less confusing, masybe show some other examples to give you more of an idea?

* `order` (enum): The canonical order. See the schema for [a list of enums](https://github.com/mdn/data/blob/master/css/syntaxes.schema.json#L234).

-- It's not very obvious what this is, from this definition; can you expand this a bit? Also, is this the right link? The linked file doesn't seem to contain a list of enums for order.

* `alsoAppliesTo` (enum): To which elements the property also applies to. See the schema for [a list of enums](https://github.com/mdn/data/blob/master/css/syntaxes.schema.json#L222)

-- again, is this the correct link?

-- GENERAL COMMENT - the property descriptions section is in between the sample short-hand and long-hand structures, which is a bit odd, as they apply to both. It almost looks like they only apply to the long-hand structure. Why don't you put both samples first, and then property descriptions afterwards?

## syntaxes.md

Looks somewhat unfinished, with a placeholder. I'm assuming you are aware of this, but mentioning just in case. Also, it contains exactly the same intro paragraph as types.md:

[CSS basic data types](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Types) define the kinds of values (keywords and units) accepted by CSS properties and functions.

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
@Elchi3 present for you ;-)

**Comment by Elchi3** [comment]
## properties.md

> percentages(enum or array of property names): 
> -- I find these definitions a bit confusing. Percentages are percentages. The value in the example above is "no" - how does a percentage refer to "no"? Could we try to rephrase this to make it a bit less confusing, masybe show some other examples to give you more of an idea?

- Good point. I think "no" is a special value saying that no percentages can be used with this property. The other enums like "referToBorderBox" are indicating where the percentage is calculated from.

> order (enum): 
>  It's not very obvious what this is, from this definition; can you expand this a bit? Also, is this the right link? The linked file doesn't seem to contain a list of enums for order.

- The correct link is https://github.com/mdn/data/blob/master/css/properties.schema.json#L235
- I have no idea about CSS, tbh. I think canonical order should be explained on MDN, but seems like it currently isn't. We could point to https://stackoverflow.com/questions/28963536/what-does-canonical-order-mean-with-respect-to-css-properties but we should really document this on MDN and then reference it here.

> alsoAppliesTo (enum):

- The correct link is https://github.com/mdn/data/blob/master/css/properties.schema.json#L153

> GENERAL COMMENT - the property descriptions section is in between the sample short-hand and long-hand structures, which is a bit odd, as they apply to both. It almost looks like they only apply to the long-hand structure. Why don't you put both samples first, and then property descriptions afterwards?

- Yes, either first the description and then examples or examples first ("on top" :laughing:). Right now it is indeed weird to have it like this.

## syntax.md
> Looks somewhat unfinished, with a placeholder. I'm assuming you are aware of this, but mentioning just in case. Also, it contains exactly the same intro paragraph as types.md:

- Yes, I didn't put much love into it as we wanted change the structure slightly. Also #41 is merged now, we need to update the example.

## types.md
- #81 is merged now, we need to update it accordingly.

File: css/at-rules.md [hunk 1]
```diff
@@ -19,18 +19,20 @@ A simple at-rule object might look like this:
...
-* `groups` (array of strings): CSS is organized in modules like "CSS Fonts" or "CSS Animations". MDN organizes features in these groups as well.
+The 3 properties seen above are all required:
+
+* `syntax` (string): This is the formal syntax of the at-rule and is usually found in the specification.
+* `groups` (array of strings): CSS is organized in modules like "CSS Fonts" or "CSS Animations". MDN organizes features in these groups as well — `groups` should contain the name of the module(s) the at-rule is defined in.
 * `status` (enum string): This is either `standard`, `nonstandard`, or `experimental` depending on the standardization status of the feature.
 
 There are 2 more properties that are optional:
-* `interfaces` (array of strings): These are the [CSSOM](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Object_Model) interfaces that belong the the at-rule. 
+* `interfaces` (array of strings): These are the [CSSOM](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Object_Model) interfaces that belong the the at-rule.
```
Comment (Elchi3) [inline css/at-rules.md]: belong to the

**Comment by Elchi3** [comment]
Do you want to fix the things we discussed in this PR or do we want them to be in a follow-up? This current diff here looks good to go to me, but we identified some more things in our discussion.

**Comment by chrisdavidmills** [comment]
So, I've responded to your comment by fixing all the points made including adding a glossary definition for canonical order, and improving the description of percentages.

I also updated the types.md to remove the stuff about angle brackets

The only thing I didn't do was update syntax.md; I think that's a bit more involved so should be rolled out to its own task.
